test/transformations.jl

```diff
@@ -1,66 +1,182 @@
 using StatsBase
-import StatsBase: transform, reconstruct
+import StatsBase: transform, reconstruct, transform!, reconstruct!
 using Statistics
 using Test
 
 @testset "Transformations" begin
+    # matrix
     X = rand(5, 8)
+    X_ = copy(X)
 
-    t = fit(ZScoreTransform, X; center=false, scale=false)
+    t = fit(ZScoreTransform, X, dims=1, center=false, scale=false)
     Y = transform(t, X)
     @test isa(t, AbstractDataTransform)
     @test isempty(t.mean)
```

Comment:
@nalimilan I noticed that tests are covered and asked to be empty. I tried to change to `zeros(T, 1)`, then transformation and reconstruct failed.

Reply:
OK, good.